<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmMat</class>
 <widget class="QWidget" name="ConfirmMat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>25</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidgetData">
       <column>
        <property name="text">
         <string>原料名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>原料编码</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>数量</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>单位</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QCheckBox" name="checkBoxAllSelect">
         <property name="text">
          <string>全选</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res.qrc">
           <normaloff>:/image/image/全选.png</normaloff>:/image/image/全选.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelContent">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonConfirm">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string> 确认采购</string>
         </property>
         <property name="icon">
          <iconset resource="../../../res.qrc">
           <normaloff>:/image/image/确认.png</normaloff>:/image/image/确认.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../res.qrc"/>
 </resources>
 <connections/>
</ui>
